main.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'app'
+
+def display_options
+  puts '---------------------------------'
+  puts 'Select an option by choosing the number:'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List rentals for a person'
+  puts '7. Quit'
+  puts '---------------------------------'
+end
+
+def handle_option(option, app)
+  option_actions = {
+    1 => -> { app.list_all_books },
+    2 => -> { app.list_all_people },
+    3 => -> { app.create_person },
+    4 => -> { app.create_book },
+    5 => -> { app.create_rental },
+    6 => -> { app.list_all_rentals },
+    7 => -> { puts 'Exiting' },
+    default: -> { puts 'Enter a number between 1 and 7.' }
+  }
+
+  action = option_actions[option] || option_actions[:default]
+  action.call
+end
+
+def main
+  app = App.new
+  @option = 0
+
+  until @option == 7
+    display_menu
```

Comment:
- [ ] I get an error when trying to run this file because this method does not exist. Please make sure that when a user/reviewer runs `main.rb`, everything should be working as expected. 
![image](https://github.com/techmoves/school-library/assets/45460192/995f2ba4-bf64-4c54-a655-99961bcc9fda)

- [ ] I commented out the `display_menu` method and ran `main.rb` and did not see the UI from the [original design](https://www.youtube.com/watch?v=vkkgrhD6aXQ) as requested. I saw this as seen in the image below
![image](https://github.com/techmoves/school-library/assets/45460192/c5051c14-7f46-4bca-82c4-5e33a2b5fa26)
I am not sure if the UI is supposed to come from the `display_menu` method.